src/main/java/Duke.java

```diff
@@ -21,6 +83,11 @@ private static void IntroductoryMessage() {
         lineBreak();
```

Comment:
I think this may cause a finite loop????

Reply:
how so?